packages/ffe-searchable-dropdown-react/src/SearchableDropdown.tsx

```diff
@@ -386,9 +388,16 @@ function SearchableDropdownWithForwardRef<Item extends Record<string, any>>(
                 >
                     {isLoading ? (
                         <Spinner />
+                    ) : isInputEmpty ? (
+                        <Icon
```

Comment:
Kan det vare slik  `<Icon fileUrl={isInputEmpty ? a : b}` /> så slipper man neste tenaries. Dette går ju kun alltså hvis ikonerna har samme størelse

Reply:
Takk for innspill! Lagt det opp på tilsvarende måte!